lib/ethpy/ethpy/hyperdrive/interface.py

```diff
@@ -175,7 +236,7 @@ def get_hyperdrive_market(web3: Web3, hyperdrive_contract: Contract) -> Hyperdri
         market_state=market_state,
         position_duration=elftime.StretchedTime(
             days=FixedPoint(pool_config["positionDuration"]) / FixedPoint(86_400),
-            time_stretch=FixedPoint(pool_config["timeStretch"]),
+            time_stretch=FixedPoint(pool_config["invTimeStretch"]),  # inverted from what solidity returns
```

Comment:
comment is weird.  invTimeStretch should be the unchanged solidity value.  timeStretch is changed to be normal value.